tests/mocha/event_block_delete_test.js

```diff
@@ -34,8 +34,10 @@ suite('Block Delete Event', function() {
       testBlock.dispose();
 
       const deleteClass = eventUtils.get(eventUtils.BLOCK_DELETE);
-      chai.assert.isTrue(spy.calledOnce);
-      chai.assert.isTrue(spy.getCall(0).args[0] instanceof deleteClass);
+      setTimeout(function() {
```

Comment:
Does it not work to pass `fireEventsNow: false` in the options, and then use `this.clock.tick()` instead of setting the timeout? This approach works but using the fake clock tick would be more standard imo.

Reply:
Using the clock tick doesn't actually test what I want it to test. If I used the clock tick, we get a false positive with the old "fix" for the original bug. This makes sure that the event listener actually gets called with the delay.

Reply:
even if "fire events immediately" is false? that surprises me but this is fine too.

Reply:
Nah nah you're right. I thought about it more after I posted my reply, and the issue is ticking immediately after `fire` is called, not ticking in general.